tools/_rapids-get-pr-artifact-github

```diff
@@ -20,6 +20,8 @@ if [[ "${package_format}" = "wheel" && -z "${RAPIDS_PY_WHEEL_NAME:+placeholder}"
     exit 1
 fi
 
+source rapids-prompt-local-github-auth
```

Comment:
Yes! I came to this conclusion too: https://github.com/rapidsai/gha-tools/issues/171#issuecomment-2864295152

Reply:
Yeah! Everywhere that we invoke `gh` in a tool that might be used interactively, this should be called.

The `if gh auth status` makes it idempotent (ensures you aren't put through the interactive login workflow multiple times).